quizapp/src/main/java/billennium/quizapp/config/SwaggerConfig.java

```diff
@@ -0,0 +1,25 @@
+package billennium.quizapp.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+@Profile("dev")
+public class SwaggerConfig {
+
+    @Bean
+    public Docket api(){
+        return new Docket(DocumentationType.SWAGGER_2)
+                .select()
+                .apis(RequestHandlerSelectors.any())
+                .paths(PathSelectors.any())
```

Comment:
tutaj bardziej do zastanowienia  może kiedyś do zmiany:) 'any' to domyslny  może tuteż wrzucić prywatną metodę 
która przyjmuję predykaty.   Możemy też dodać ApiInfoBuildera gdzie możemy ustawić info na temat api. Ogólnie API Swaggera jest bardzo rozbudowane. I często konfiguracja nie jest trywialna.